### PR TITLE
Make AWSOrganizationsNotInUseException non-fatal

### PR DESCRIPTION
**Pull request by ajrosen**

### CODE REVIEW

File: lib/aws_recon/collectors/organizations.rb [hunk 1]
```diff
@@ -13,13 +13,18 @@ def collect
...
 
-      struct = OpenStruct.new(response.organization.to_h)
-      struct.type = 'organization'
+        struct = OpenStruct.new(response.organization.to_h)
+        struct.type = 'organization'
 
-      resources.push(struct.to_h)
+        resources.push(struct.to_h)
+      end
+    rescue Aws::Organizations::Errors::AWSOrganizationsNotInUseException => e
```
Comment (joshlarsen) [inline lib/aws_recon/collectors/organizations.rb]: Add `AWSOrganizationsNotInUseException` to private `suppressed_errors` array.

File: lib/aws_recon/collectors/organizations.rb [hunk 2]
```diff
@@ -13,13 +13,18 @@ def collect
...
+      log_error(e.code)
+      return resources
```
Comment (joshlarsen) [inline lib/aws_recon/collectors/organizations.rb]: Optionally suppress or raise, as in [L57](https://github.com/darkbitio/aws-recon/blob/729eebfb47f4ca0a92f72069f2e8d181c8a28ed1/lib/aws_recon/collectors/organizations.rb#L57)

**Review by joshlarsen — CHANGES_REQUESTED** [review]

**Comment by joshlarsen** [comment]
@ajrosen thanks for the PR. Could you please adjust slightly per the review comments? See [example](https://github.com/darkbitio/aws-recon/blob/main/lib/aws_recon/collectors/organizations.rb#L39-L61).

Thanks!

**Review by ajrosen — COMMENTED** [review]
I tested with and without -q for both _AWSOrganizationsNotInUseException_ and _AccessDeniedException_.

Thanks!

**Review by joshlarsen — APPROVED** [review]
Thanks!

**Pull request merged**